test/unit/format.js

```diff
@@ -134,6 +134,22 @@ describe('format API', () => {
             expect(formatDate(timestamp)).toBe(df.format(timestamp));
         });
 
+        it('uses the time zone specified by the provider', () => {
+            const timestamp = Date.now();
+            config = {...config, timeZone: 'Pacific/Wake'};
```

Comment:
There's no need to spread `...config` here since you're going to mutate `config` anyways.
```js
config.timeZone = 'Pacific/Wake';
```